Cargo.toml

```diff
@@ -92,7 +92,7 @@ sqlx = { version = "0.7.3", features = ["runtime-tokio-native-tls", "postgres",
 wiremock = "0.5.19"
 itertools = "0.11.0"
 sha3 = "0.10.8"
-validator = { version = "0.16.1", features = ["derive"] }
+validator = { version = "0.17.0", features = ["derive"] }
```

Comment:
Upgrade to support custom validators accepting `Option<T>` instead of `T`, allowing the ability to validate the state of the `Option` and not just `T`.

Some may consider this change [a bug](https://github.com/Keats/validator/issues/310)